### PR TITLE
GetFile and GetFileDirectURL method to fetch url of file received from user

### PR DESCRIPTION
**Pull request by brainfucker**

### CODE REVIEW

File: bot.go [hunk 1]
```diff
@@ -620,3 +620,40 @@ func (b *Bot) AnswerCallbackQuery(callback *Callback, response *CallbackResponse
 
 	return nil
 }
+
+// Get file object with FilePath field
```
Comment (tucnak) [inline bot.go]: Not self-explanatory enuff.

File: bot.go [hunk 2]
```diff
@@ -620,3 +620,40 @@ func (b *Bot) AnswerCallbackQuery(callback *Callback, response *CallbackResponse
...
+	}
+
+	if !responseRecieved.Ok {
+		return File{}, fmt.Errorf("telebot: %s", responseRecieved.Description)
+	}
+
+	return responseRecieved.Result, nil
+}
+
+// Get file url string
```
Comment (tucnak) [inline bot.go]: Same here.

**Comment by tucnak** [comment]
Do you have any desire of adding documentation? The thing is I want this in-tree ASAP, so in case if you can't really do it, I'd love to finish the job on my own (again, unless you wanna do this.)

**Comment by brainfucker** [comment]
Just updated docs, feel free to change it any way you want, if the way I made this functions doesn't fit your design idea

**Comment by tucnak** [comment]
Fanks!

**Pull request merged**